### PR TITLE
added shippable.yml

### PR DESCRIPTION
**Pull request by ragsden**
fixes https://github.com/aye0aye/deploy/issues/1
fixes https://github.com/aye0aye/deploy/issues/2

### CODE REVIEW

**Comment by ragsden** [comment]
I can look more into variables, if you want me to.

**Comment by ragsden** [comment]
`````
export TF_VAR_key_name=
export TF_VAR_access_key=
export TF_VAR_secret_key=
terraform apply
````

File: shippable.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+infra:
+    clusters:
+        aye-aye:
```
Comment (trriplejay) [inline shippable.yml]: i think we want to have a `-` here to make it an array of clusters, like this:
```
infra:
    clusters:
      - aye-aye:
            provider: ECS
            type: terraform
```

**Comment by avinci** [comment]
sorry my bad. I merged it. We should add the change @trriplejay wanted

**Pull request merged**